content/articles/aes-rsa-encryption/index.md

```diff
@@ -0,0 +1,138 @@
+
+### Table of contents
+1. [Prerequisites](#prerequisites)
+2. [What is an encryption algorithm?](#what-is-an-encryption-algorithm)
+3. [Getting started with AES encryption algorithm](#getting-started-with-aes-encryption-algorithm)
+4. [Functionality of AES algorithm](#functionality-of-aes-algorithm)
+5. [Getting started with RSA encryption algorithm](#getting-started-with-rsa-encryption-algorithm)
+6. [RSA algorithm functionality](#rsa-algorithm-functionality)
+7. [Conclusion](#conclusion)
+
+### Prerequisites
+
+For a reader to understand this article, they should have the following prior knowledge:
+
+- A basic understanding of the concept of data encryption and the need for encryption in data communication.
+- A beginner's understanding of data cryptography.
+
+### What is an encryption algorithm?
+
+An encryption algorithm is a set of rules used to transform data into ciphertext using an encryption key. This will make the data appear to be completely different from the initial data and will seem random. The cipher will then be transmitted safely to a recipient who will then go ahead and recover the initial data through a process called decryption using a decryption key.
+
+AES and RSA are an example of these algorithms that are used in data encryption in data communication. AES is an acronym that stands for Advanced Encryption Standard while RSA stands for Rivest, Shamir, Adleman.
+
+### Getting started with AES encryption algorithm
+
+It was initially referred to as Rijndael and is a Symmetric [block cipher](https://en.wikipedia.org/wiki/Block_cipher) algorithm. AES was specifically developed for encrypting electronic data back in the United States by the [NIST](https://en.wikipedia.org/wiki/National_Institute_of_Standards_and_Technology). Ever since the AES has been used worldwide in encrypting confidential data.
+
+AES was developed as a replacement for [DES](https://searchsecurity.techtarget.com/definition/Data-Encryption-Standard) after it started showing vulnerabilities to Brute-force attackers.
+
+### Functionality of AES algorithm
+
+The functionality of the AES algorithm might seem complex but in a real sense, it is really simple to understand. The first step is knowing that AES has 3 Block ciphers, namely:
+
+- **AES-128**
+
+As we already know, the encryption and decryption processes are done using a key.
+This block cipher uses a key that is 128 bits long for encryption and decryption of a message.
+- **AES-192**
+
+The AES-192 uses a key that is 192 bits long for the encryption and decryption of messages.
+
+- **AES-256**
+
+The AES-256 uses a key that is 256 bits long for the encryption and decryption of messages.
```

Comment:
Add a sentence or two more that add to the differences between these three block ciphers. I'm sure the number of bits isn't the only difference